projects/extension/sql/idempotent/008-embedding.sql

```diff
@@ -41,6 +59,8 @@ begin
     case _implementation
         when 'openai' then
             -- ok
+        when 'ollama' then
+            -- ok
```

Comment:
What kind of error are we trying to catch here? The `ai.embedding_*` functions define mandatory arguments, so it will ensure that we have _some_ values. The configuration function may not turn those into a json which actually contains the right fields, but I'm not sure that we should be checking that here.

Reply:
It may be overkill, but since these values are jsonb, someone could try to hack together a config without using the appropriate configuration functions.